packages/ocs/dashboards/persistent-internal/status-card/status-card.tsx

```diff
@@ -127,7 +129,7 @@ const cephClusterResource = {
   isList: true,
 };
 
-export const StatusCard: React.FC = () => {
+const StatusCard: React.FC<{}> = () => {
```

Comment:
```suggestion
const StatusCard: React.FC = () => {
```

Reply:
@weirdwiz just an observation: if you find out later you need to pass some prop to Card (provided this is wrapping the Card component):
```suggestion
const StatusCard: React.FC<CardProps> = () => {
```